prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -92,6 +92,11 @@ spec:
               cpu: 10m
           securityContext:
             privileged: true
+          env:
+          - name: MY_NODE_NAME
```

Comment:
From that point of view, I think `NODE_NAME` is better.
The value is actually available in `$HOSTNAME`, for a pod in the host namespace, but I was afraid that was too obscure.  But today it doesn't seem so bad.